### DISCUSSION TITLE
Multiple Users DN

### DISCUSSION THREAD

**Discussion by mr-nothing** [keycloak/keycloak#32865] [Q&A]
Hey folks!
First of all, thanks for great product!
Now let me please describe the issue I faced:
- I have 2 different OU `OU=Special Users,DC=organization,DC=my` and `OU=Ordinary Users,DC=organization,DC=my` in my LDAP which I'd like to sync to my keycloak instance
- Now I faced the restriction in keycloak interface that allows only one Users DN to search in

So the question: is there any way I can configure this, some workarounds maybe? Or the only way I can configure this is to have the only root OU for all the users I'd like to sync to my keycloak?

Thanks a lot in advance for your help!

---

keycloak version 25.0.4

**Comment by SaschaWeisenbergerMVV** [comment]
seems like adding a second ldap provider with a different users DN does the trick.
